src/lib.rs

```diff
@@ -544,19 +544,22 @@ impl Client {
         let job = self
             .http_client
             .request(request)
-            .map(|res| Some(ResponseStatus::from(res.status())))
-            .map_err(|error| {
-                println!("Error while sending a report to Rollbar.");
-                print!("The error returned by Rollbar was: {:?}.\n\n", error);
-
-                None::<ResponseStatus>
-            });
+            .map(|result|
+                match result {
+                    Ok(resp) => Some(ResponseStatus::from(resp.status())),
+                    Err(error) => {
+                        println!("Error while sending a report to Rollbar.");
+                        print!("The error returned by Rollbar was: {:?}.\n\n", error);
+                        None::<ResponseStatus>
+                    }
+                }
+            );
 
         thread::spawn(move || {
-            current_thread::Runtime::new()
-                .unwrap()
-                .block_on(job)
-                .unwrap()
+            let runtime = tokio::runtime::Builder::new_current_thread()
+                                    .enable_io()
+                                    .build().expect("Error creating runtime");
+            runtime.block_on(job)
```

Comment:
The existing code (and still with this change) is buggy when reusing the client across threads. See https://github.com/RoxasShadow/rollbar-rs/issues/18

Should be easy enough to fix by replacing hyper with `reqwest::blocking`.